### PR TITLE
[DOCS] Add custom fields to cases

### PR DESCRIPTION
**Pull request by lcawl**
Relates to https://github.com/elastic/kibana/issues/160236

This PR updates the [Security Cases documentation](https://www.elastic.co/guide/en/security/master/cases-open-manage.html) with information about how to use custom fields.

It also fixes a link in the email notification steps (per https://github.com/elastic/kibana/pull/168656).

NOTE: This PR includes two less screenshots than I added in https://github.com/elastic/kibana/pull/169057. Those screenshots show where the custom fields appear when you create a new case and edit an existing case. If you want those screenshots here too, just lmk and I can automate and add them as well.

### CODE REVIEW

File: docs/cases/cases-manage.asciidoc [hunk 1]
```diff
@@ -37,6 +37,46 @@ NOTE: If you've selected a connector for the case, the case is automatically pus
...
+[role="screenshot"]
+image::images/cases-add-custom-field.png[Add a custom field in case settings]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+--
+
+. Enter a field label.
+
+. Choose a field type: text or toggle.
+
+. If you want the text field to be mandatory in all cases, select *Make this field required*.
```
Comment (nastasha-solomon) [inline docs/cases/cases-manage.asciidoc:70]:
```suggestion
. (Optional) To make the text field mandatory, select *Make this field required*.
```

File: docs/cases/cases-manage.asciidoc [hunk 2]
```diff
@@ -37,6 +37,46 @@ NOTE: If you've selected a connector for the case, the case is automatically pus
...
+
+. Click *Save field*.
+
+You can subsequently remove or edit custom fields on the *Settings* page.
```
Comment (nastasha-solomon) [inline docs/cases/cases-manage.asciidoc:74]: I kinda want to bring the next three lines together so they look less disjointed. Here's an option:

```suggestion
After you create custom fields, they're added to all new and existing cases. Existing cases have null values for new text fields until you set them in. To manage custom fields, go the *Settings* page and remove or edit them.
```

**Review by nastasha-solomon — APPROVED** [review]
Looks great! Just left two suggestions for your consideration. Thank you for taking care of this!

File: docs/cases/cases-manage.asciidoc (hunk 2 above)
Comment (lcawl) [reply docs/cases/cases-manage.asciidoc]: Fixed in f0bb1b1e12afa0e2583223dce275bf58fce07103, thanks!

**Pull request merged**